tests/light/functional_tests/destination_drivers/clickhouse_destination/test_clickhouse_connection_options.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2025 Axoflow
+# Copyright (c) 2025 Andras Mitzki <andras.mitzki@axoflow.com>
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+import uuid
+
+import pytest
+from axosyslog_light.driver_io.clickhouse.clickhouse_io import ClickhouseClient
+
+clickhouse_options = {
+    "database": "default",
+    "table": "test_table",
+    "user": "default",
+    "password": "'password'",
+    "schema": '"message" "String" => "$MSG"',
+}
+
+
+def test_clickhouse_destination_valid_options_db_run(request, config, syslog_ng, clickhouse_server):
+    custom_input_msg = f"test message {str(uuid.uuid4())}"
+    generator_source = config.create_example_msg_generator_source(num=1, template=f'"{custom_input_msg}"')
+    clickhouse_destination = config.create_clickhouse_destination(**clickhouse_options)
+    config.create_logpath(statements=[generator_source, clickhouse_destination])
+
+    clickhouse_server.start()
+    clickhouse_destination.create_table(clickhouse_options["table"], [("message", "String")])
+    request.addfinalizer(lambda: clickhouse_destination.delete_table())
+
+    syslog_ng.start(config)
+
+    assert clickhouse_destination.get_stats() == {'msg_size_avg': 52, 'eps_last_1h': 0, 'processed': 1, 'eps_last_24h': 0, 'batch_size_max': 52, 'memory_usage': 0, 'dropped': 0, 'queued': 0, 'written': 1, 'eps_since_start': 0, 'msg_size_max': 52, 'batch_size_avg': 52}
```

Comment:
I would hate to modify these test cases every time a value gets added or removed in the stats output. Can we use the `>` operator to check only the relevant parts instead?

Reply:
fixed